2024/political-donations-top-pacs/main.js

```diff
@@ -0,0 +1,80 @@
+topPACs = [
+    { name: 'Biden For President/<br/>Biden Victory Fund', y: 367971 },
+    { name: 'ActBlue', y: 308475 },
+    { name: 'Democratic National<br/>Committee', y: 48787 },
+    { name: 'DCCC', y: 35841 },
+    { name: 'Lauren Underwood<br/>For Congress', y: 35644 }
+];
+
+yAxisLabels = [0, 25000, 50000, 100000, 200000, 300000, 400000];
+
+Highcharts.setOptions({
+    lang: {
+        thousandsSep: ','
+    }
```

Comment:
You can move `lang` into the chart options below, yeah?

Reply:
i think it has to be globally set `Language object. The language object is global and it can't be set on each chart initialization. Instead, use Highcharts.setOptions to set it before any chart is initialized`

https://api.highcharts.com/highcharts/lang